sentry/resource_sentry_project.go

```diff
@@ -187,12 +203,31 @@ func resourceSentryProjectRead(ctx context.Context, d *schema.ResourceData, meta
 		"org":         org,
 	})
 
+	setTeams := func() error {
+		if len(proj.Teams) <= 1 || proj.Teams == nil {
```

Comment:
`len(nil) = 0` no need for `proj.Teams == nil`.

Reply:
Nice! Thanks for this! Fixed in https://github.com/Canva/terraform-provider-sentry/pull/33/commits/144799b773b30e9898efd2e41ad05a6ce1b299d2. Learn new things everyday